recipe/data_dir.patch

```diff
@@ -0,0 +1,11 @@
+--- pyproj-1.9.5.1rel.orig/setup.py	2016-01-06 20:26:39.000000000 -0200
++++ pyproj-1.9.5.1rel/setup.py	2018-08-31 14:09:52.990151172 -0300
+@@ -24,7 +24,7 @@
+     runtime_library_dirs=libdirs,libraries=libraries)
+ 
+     # over-write default data directory.
+-    pyproj_datadir = os.path.join(os.path.join(proj_dir,'share'),'proj')
++    pyproj_datadir = os.environ['PROJ_LIB']
```

Comment:
@msarahan for some reason this is not working on Windows even though this variable is defined in `proj4`'s activation scripts.. Not sure what is wrong here :-/ Maybe I'm missing something obvious.